pkg/ast/ast.go

```diff
@@ -124,18 +128,59 @@ type ContextCondition struct {
 	Subject Subject
 	Where   *WhereClause
 }
-type ContextActionRule struct {
-	Context *ContextStatement
 
+func (a *ContextCondition) String() string {
+	var out bytes.Buffer
+	if !types.IsNilInterface(a.Subject) {
+		out.WriteString(fmt.Sprintf("subject group %s ", a.Subject.String()))
```

Comment:
similar to ActionStatement
```suggestion
		out.WriteString(a.Subject.String() + " ")
```

Reply:
thx! done.